<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>80</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Sort By Rating:</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HeadingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cinemas Best Benchmark V1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Generate New List</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Credits">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>700</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BSTTimeLabel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>560</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BST Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxHeapTimeLabel">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>620</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Heap Time:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>871</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <zorder>tableWidget</zorder>
   <zorder>SortButton</zorder>
   <zorder>HeadingLabel</zorder>
   <zorder>GenerateButton</zorder>
   <zorder>Credits</zorder>
   <zorder>BSTTimeLabel</zorder>
   <zorder>MaxHeapTimeLabel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
